### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by fymbc**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Financial Assurance Revolutionary Telemarketer (FART) is a **desktop app for man
 
    * `clear` : Deletes all contacts.
 
-   * `find john`  : Displays all contacts with keyword(s) matching "john"
+   * `find n/ john`  : Displays all contacts with name(s) matching "john"
```
Comment (AlphaPyke) [inline docs/UserGuide.md:47]: I believe there should be no " "?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -130,23 +130,19 @@ Examples:
...
 Examples:
-* `find John` returns `john` and `John Doe`
-* `find alex david` returns `Alex Yeoh`, `David Li`<br>
-  ![result for 'find alex david'](images/findAlexDavidResult.png)
-* `find serangoon` returns ``Bernice Yu``, ``David Li``
+* `find n/John` returns clients with names `john` and `John Doe`.
+* `find a/serangoon` returns ``Bernice Yu``, ``David Li``.
 ![result for 'find serangoon'](images/findSerangoonResult.png)
 
 ### Deleting a person : `delete`
```
Comment (AlphaPyke) [inline docs/UserGuide.md:148]: Perhaps we could change this to client as well?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -257,14 +260,14 @@ _Details coming soon ..._
...
-**Delete** | `delete INDEX`<br> e.g., `delete 3`
-**Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
-**Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
-**List** | `list`
-**Help** | `help`
-**Paid** | `paid INDEX f/FREQUENCY`<br> e.g., `paid 3 f/ 3`
-**Unpaid** | `unpaid INDEX`<br> e.g., `unpaid 3`
+| Action     | Format, Examples                                                                                                                                                                              |
+|------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Add**    | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS b/BIRTHDAY [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 b/11 09 2001 t/friend t/colleague` |
```
Comment (AlphaPyke) [inline docs/UserGuide.md:265]: I think we should change the phone number in this example to a valid one

Comment (AlphaPyke) [reply docs/UserGuide.md:265]: In addition, the a/ADDRESS can be bracketed as well now that address is optional?

**Review by AlphaPyke — COMMENTED** [review]
Great job updating the UG with many of the new features, just some minor details to touch up!

File: docs/UserGuide.md (hunk 1 above)
Comment (fymbc) [reply docs/UserGuide.md:47]: Thanks! Will fix

File: docs/UserGuide.md (hunk 2 above)
Comment (fymbc) [reply docs/UserGuide.md:148]: Currently only changing the commands I uploaded. Let's update again in the next version

File: docs/UserGuide.md (hunk 3 above)
Comment (fymbc) [reply docs/UserGuide.md:265]: Noted! Will fix

**Pull request merged**